### ISSUE TITLE
Perssisted focused property with v2.1.0-alpha2 on Firefox

### ISSUE THREAD

**Issue by abdonrd** [vaadin/vaadin-button#98]
The `vaadin-button` doesn't drop the `focused` property when it lose the focus.
Using the `v2.1.0-alpha2` from npm.

This demo just adding the `vaadin-button` to a new PWA Starter Kit project:

![jul-06-2018 19-41-10](https://user-images.githubusercontent.com/1007051/42392859-96ad5fd8-8154-11e8-8754-997f908d4d31.gif)

**Comment by abdonrd** [comment]
Same thing with `focus-ring` property.

**Comment by web-padawan** [comment]
Confirmed the issue in Firefox, live demo: 
https://polymer3-webpack-starter.herokuapp.com/employee-new

**Comment by abdonrd** [comment]
Thanks @web-padawan! I was checking that everything was fine before publishing a website and I just found myself with this problem.

**Comment by web-padawan** [comment]
Note: confirmed the issue with Firefox 61 and it persists after updating to 61.0.1

Still investigating but it seems something goes wrong with the `focusout` event there. I have only found this bug, although it might not be related: https://bugzilla.mozilla.org/show_bug.cgi?id=1472887

**Comment by abdonrd** [comment]
I also try with Firefox 52.9.0 ESR (don't ask 🙈 haha): same behavior.

**Comment by web-padawan** [comment]
Hmm, I'm unsure which changes are backported to ESR and what is a schedule there. 
The problem there might be different, AFAIK `focusout` support has been shipped in 52 stable, but ESR might certainly have lots of updates since that time. We are not running tests against Firefox ESR currently, but once we fix it for latest version, it would be worth checking.

**Comment by abdonrd** [comment]
Now with https://github.com/vaadin/vaadin-control-state-mixin/issues/40 fixed and [vaadin-control-state-mixin v2.1.0](https://github.com/vaadin/vaadin-control-state-mixin/releases/tag/v2.1.0) released... Can I expect a new version of `vaadin-button` soon? 🙏

**Comment by limonte** [comment]
> Now with vaadin/vaadin-control-state-mixin#40 fixed and vaadin-control-state-mixin v2.1.0 released... Can I expect a new version of vaadin-button soon?

`bower update` will handle new versions properly.

```sh
 bower install vaadin-button#2.1.0-alpha3 
bower vaadin-button#2.1.0-alpha3           cached https://github.com/vaadin/vaadin-button.git#2.1.0-alpha3
bower vaadin-button#2.1.0-alpha3         validate 2.1.0-alpha3 against https://github.com/vaadin/vaadin-button.git#2.1.0-alpha3
bower vaadin-control-state-mixin#^2.1.0-alpha2           cached https://github.com/vaadin/vaadin-control-state-mixin.git#2.1.0
bower vaadin-control-state-mixin#^2.1.0-alpha2         validate 2.1.0 against https://github.com/vaadin/vaadin-control-state-mixin.git#^2.1.0-alpha2
bower vaadin-material-styles#^1.1.0-alpha2               cached https://github.com/vaadin/vaadin-material-styles.git#1.1.0-alpha2
bower vaadin-material-styles#^1.1.0-alpha2             validate 1.1.0-alpha2 against https://github.com/vaadin/vaadin-material-styles.git#^1.1.0-alpha2
bower vaadin-lumo-styles#^1.1.0-alpha3                   cached https://github.com/vaadin/vaadin-lumo-styles.git#1.1.0-alpha3
bower vaadin-lumo-styles#^1.1.0-alpha3                 validate 1.1.0-alpha3 against https://github.com/vaadin/vaadin-lumo-styles.git#^1.1.0-alpha3
bower vaadin-element-mixin#^1.1.0                        cached https://github.com/vaadin/vaadin-element-mixin.git#1.1.1
bower vaadin-element-mixin#^1.1.0                      validate 1.1.1 against https://github.com/vaadin/vaadin-element-mixin.git#^1.1.0
bower vaadin-usage-statistics#^1.1.0-alpha1              cached https://github.com/vaadin/vaadin-usage-statistics.git#1.1.0-alpha1
bower vaadin-usage-statistics#^1.1.0-alpha1            validate 1.1.0-alpha1 against https://github.com/vaadin/vaadin-usage-statistics.git#^1.1.0-alpha1
bower vaadin-development-mode-detector#^1.1.0-alpha2     cached https://github.com/vaadin/vaadin-development-mode-detector.git#1.1.0-alpha2
bower vaadin-development-mode-detector#^1.1.0-alpha2   validate 1.1.0-alpha2 against https://github.com/vaadin/vaadin-development-mode-detector.git#^1.1.0-alpha2
bower vaadin-button#2.1.0-alpha3                        install vaadin-button#2.1.0-alpha3
bower vaadin-control-state-mixin#^2.1.0-alpha2          install vaadin-control-state-mixin#2.1.0
bower vaadin-lumo-styles#^1.1.0-alpha3                  install vaadin-lumo-styles#1.1.0-alpha3
bower vaadin-element-mixin#^1.1.0                       install vaadin-element-mixin#1.1.1
bower vaadin-usage-statistics#^1.1.0-alpha1             install vaadin-usage-statistics#1.1.0-alpha1
bower vaadin-development-mode-detector#^1.1.0-alpha2    install vaadin-development-mode-detector#1.1.0-alpha2
bower vaadin-material-styles#^1.1.0-alpha2              install vaadin-material-styles#1.1.0-alpha2

vaadin-button#2.1.0-alpha3 bower_components/vaadin-button
├── polymer#2.6.0
├── vaadin-control-state-mixin#2.1.0
├── vaadin-element-mixin#1.1.1
├── vaadin-lumo-styles#1.1.0-alpha3
├── vaadin-material-styles#1.1.0-alpha2
└── vaadin-themable-mixin#1.2.0

vaadin-control-state-mixin#2.1.0 bower_components/vaadin-control-state-mixin
└── polymer#2.6.0
```

**Comment by abdonrd** [comment]
Great, thanks @limonte!
